ClemBot.Bot/bot/cogs/tags_cog.py

```diff
@@ -68,6 +68,45 @@ async def tag(self, ctx: commands.Context, tag_name: Optional[str] = None):
                                          channel=ctx.channel,
                                          timeout=360)
 
+
+    @tag.command(aliases=['claimed'])
+    @ext.long_help(
+        'Lists all tags owned by a given user or the called user if no user is provided.'
+    )
+    @ext.short_help('Lists owned tags')
+    @ext.example(['tag owned', 'tag owned @user'])
+    # returns a list of all tags owned by the calling user or given user returned in pages
+    async def owned(self, ctx: commands.Context, user: Optional[discord.Member] = None):
+        if not user:
+            user = ctx.author
+        tags = await self.bot.tag_route.get_guilds_tags(ctx.guild.id)
+
+        ownedTags = []
+        for tag in tags:
+            if tag.user_id == user.id:
+                ownedTags.append(tag)
+        
+        if not ownedTags:
+            embed = discord.Embed(title=f'{user.display_name}\'s Tags', color=Colors.ClemsonOrange)
+            embed.add_field(name='Tags', value='User does not own any tags', inline=True)
+            embed.set_footer(text=self.get_full_name(ctx.author), icon_url=ctx.author.display_avatar.url)
+            await ctx.send(embed=embed)
+            return
+
+        tags_url = f'{bot_secrets.secrets.site_url}dashboard/{ctx.guild.id}/tags'
+        pages = self.chunked_tags(ownedTags, TAG_CHUNK_SIZE, ctx.prefix, 'Available Tags', tags_url)
+        for tag in ownedTags:
+            embed = discord.Embed(color=Colors.ClemsonOrange, title=f'{tag.name}')
+            embed.set_author(name=f'{self.bot.user.name} - Tags', url=LINK_URL, icon_url=self.bot.user.display_avatar.url)
+            embed.add_field(name='Content', value=tag.content)
+            embed.add_field(name='Uses', value=f'{tag.use_count} use{"s" if tag.use_count != 1 else ""}')
+            pages.append(embed)
```

Comment:
pages is already an array of embeds

Reply:
Yes before that loop, it is the summary view of all tags owned by a user. This loop then appends the tag contents as pages so the user can easily see the content of all their tags

Reply:
hmm, i would prefer we use the site for that but i can see the appeal

Reply:
More steps for users

Reply:
we ultimately want to push people to the site as that is where new features will be